### DISCUSSION TITLE
What is the suggested way to install certificates?

### DISCUSSION THREAD

**Discussion by ladeak** [dotnet/dotnet-docker#5037] [Q&A]
What is the suggested way to install corporate certificates as trusted root and intermediate certificates on a chiseled linux image?

What is the suggested way to maintain these over the lifetime of it? ie. if it expires, but I would not want to rebuild the application so that I can update the certs?

**Comment by lbussell** [comment]
Hi, I filed https://github.com/dotnet/dotnet-docker/issues/5045 for adding documentation for this scenario.

**Reply by ladeak** [reply]
Thank you
